tests/test_auth.py

```diff
@@ -508,6 +509,28 @@ def test_msi_vm(self):
             credentials = MSIAuthentication(resource="foo")
             assert credentials.scheme == "TokenTypeWebApp"
             assert credentials.token == json_payload
+        
+        # WebApp with User Assigned Identity
+
+        json_payload = {
+            'token_type': "TokenTypeWebApp",
+            "access_token": "AccessToken"
+        }
+        httpretty.register_uri(httpretty.GET,
+                               'http://127.0.0.1:41741/MSI/token/?resource=foo&api-version=2017-09-01&clientid=bar',
```

Comment:
There is a discussion with Arturo right now that this will be changed to client_id. Do you have any insights?

Reply:
I sure don't, sorry - wasn't aware of those discussions

I would actually love to see this be made consistent with VM IMDS, and then we simplify everything to client_id everywhere. If/when this is resolved, please do share :)

I have an open PR on the .NET side that would need the same changes (https://github.com/Azure/azure-sdk-for-net/pull/5181)